### PR TITLE
Update PRIVACY.md

### PR DESCRIPTION
**Pull request by arashd**
First draft of updates to PRIVACY.md to be released with the new telemetry and 0.4.0.

### CODE REVIEW

File: PRIVACY.md [hunk 1]
```diff
@@ -1,87 +1,113 @@
...
 
-Bento collects usage data to help inform how to improve the product. This document describes:
+Bento collects usage data to help us improve the product. This document describes:
 
-* principles that guide the collection and usage of usage data
-* the data collected
-* what data is not collected
-* why specific data is collected and how it’s used
+* the principles that guide our data collection decisions
+* the breakdown of the data that is and is not collected
```
Comment (nbrahms) [inline PRIVACY.md]:
```suggestion
* the breakdown of the data that are and are not collected
```

I know, I spent too much time in the sciences...  One datum, two data.

File: PRIVACY.md [hunk 2]
```diff
@@ -1,87 +1,113 @@
...
+* how we use data we collect to make Bento better
```
Comment (nbrahms) [inline PRIVACY.md]:
```suggestion
* how we use the data we collect to make Bento better
```

File: PRIVACY.md [hunk 3]
```diff
@@ -1,87 +1,113 @@
...
 3. **Limited data**: Collect what is needed, de-identify where possible, and delete when no longer necessary
 
 ## Collected Data
 
-Bento collects data to help improve the Bento user experience for users everywhere. The following information is collected and sent to r2c:
+Bento collects data to help improve its underlying tools and user experience. Two types of data are collected:
+
+### Usage Data
+
+As the name suggests, usage data tells us what commands you ran, what options and flags you ran them with, as well as information that helps us debug any issues users may be facing, e.g.
```
Comment (nbrahms) [inline PRIVACY.md]:
```suggestion
As the name suggests, usage data tell us what commands you ran, what options and flags you ran them with, as well as information that helps us debug any issues users may be facing, e.g.
```

File: PRIVACY.md [hunk 4]
```diff
@@ -1,87 +1,113 @@
...
+* The name of the command you ran,
+* Arguments or options,
+* How long the command took to run,
+* If the command ran in a CI environment,
+* The version of Bento you're using, 
+* What OS and shell you're running in, etc.
+
+See Description of Fields for a full list.
+
+While using Bento in Beta, we associate this data with your email address to facilitate our outreach efforts.
```
Comment (nbrahms) [inline PRIVACY.md]:
```suggestion
While using Bento in Beta, we associate these data with your email address to facilitate our outreach efforts.
```

File: PRIVACY.md [hunk 5]
```diff
@@ -1,87 +1,113 @@
...
+
+### Results Data
+
+Results data contains information about the code check rules that fired when you ran Bento, and helps us make our rules better. This includes:
```
Comment (nbrahms) [inline PRIVACY.md]:
```suggestion
Results data contain information about the code check rules that fired when you ran Bento, and help us make our rules better. These include:
```

File: PRIVACY.md [hunk 6]
```diff
@@ -1,87 +1,113 @@
...
+
+* Data on the number of times rules fired across your repo,
```
Comment (nbrahms) [inline PRIVACY.md]:
```suggestion
* The number of times rules fired across your repo,
```

File: PRIVACY.md [hunk 7]
```diff
@@ -1,87 +1,113 @@
...
+* Rules you disabled,
+* The number of results that were fixed or archived since the last run, and
+* Hashes of the repository and commit Bento produced these results for,
```
Comment (nbrahms) [inline PRIVACY.md]:
```suggestion
* Hashes of the repository and commit for which Bento produced these results,
```

File: PRIVACY.md [hunk 8]
```diff
@@ -1,87 +1,113 @@
...
 
-* Anonymized version of repository name, current commit, paths of files, names of checks that fire
-* Aggregate count of checks that fire
-* Names of checks that are disabled for the tools running in Bento
-* The user’s OS, shell information, user agent, and anonymized IP
-* Bento asks users to share error dump data on crashes
-* User’s email address (if the user provides it)
+In addition, we collect a hashed version of the repo and commit Bento was run on. We consider this information more sensitive and do NOT associate it with user-identifiable information.
```
Comment (nbrahms) [inline PRIVACY.md]: The antecedent of "this information" is confusing. Do you mean that we consider the repo hash to be sensitive, or do we consider the results data to be sensitive?

File: PRIVACY.md [hunk 9]
```diff
@@ -1,87 +1,113 @@
...
 
 ### Data NOT collected
 
-Bento minimizes the amount of personal data collected or shared, and limits that only to the amount necessary to support product operation. The following data is not collected and not sent to r2c. It does not leave your computer and is not sent or shared with anyone.
+We try hard to balance our desire to collect data for improving Bento and its underlying tools with our users' need for privacy. It does not leave your computer and is not sent or shared with anyone.
```
Comment (nbrahms) [inline PRIVACY.md]: "try hard" seems weird to me. What about "We strive"?

Comment (nbrahms) [reply PRIVACY.md]: What is the "It" that does not leave your computer?

File: PRIVACY.md [hunk 10]
```diff
@@ -1,87 +1,113 @@
...
 # Bento Privacy Policy
 
-Bento collects usage data to help inform how to improve the product. This document describes:
+Bento collects usage data to help us improve the product. This document describes:
 
-* principles that guide the collection and usage of usage data
-* the data collected
-* what data is not collected
-* why specific data is collected and how it’s used
+* the principles that guide our data collection decisions
```
Comment (nbrahms) [inline PRIVACY.md]:
```suggestion
* the principles that guide our data-collection decisions
```

File: PRIVACY.md (hunk 9 above)
Comment (arashd) [reply PRIVACY.md]: woops

**Pull request merged**